src/platform_impl/macos/event_loop.rs

```diff
@@ -125,6 +126,16 @@ impl<T: 'static> EventLoopWindowTarget<T> {
     set_progress_indicator(progress);
   }
 
+  #[inline]
+  pub fn set_badge_count(&self, count: Option<i64>, _desktop_filename: Option<String>) {
+    set_badge_label(count.and_then(|count| Some(format!("{}", count))));
+  }
+
+  #[inline]
+  pub fn set_badge_label(&self, label: Option<String>) {
+    set_badge_label(label);
+  }
```

Comment:
looks like this is not used

Reply:
Yep, this has been reserved, I shall add it as a macos specific api